openpdf/src/main/java/com/lowagie/text/pdf/AcroFields.java

```diff
@@ -162,7 +162,11 @@ void fill() {
     }
     for (int k = 1; k <= reader.getNumberOfPages(); ++k) {
       PdfDictionary page = reader.getPageNRelease(k);
-      PdfArray annots = (PdfArray) PdfReader.getPdfObjectRelease(page.get(PdfName.ANNOTS), page);
+      Object o = PdfReader.getPdfObjectRelease(page.get(PdfName.ANNOTS), page);
+      PdfArray annots = null;
+      if(o instanceof PdfArray) {
+         annots =  (PdfArray) PdfReader.getPdfObjectRelease(page.get(PdfName.ANNOTS), page);
```

Comment:
Can you cast "o" instead of calling the methods again?
like
`annots =  (PdfArray)  o;`

Reply:
Absolutely, I will fix it.